code/dst-explore.py

```diff
@@ -0,0 +1,133 @@
+# SetupProject DaVinci v36r6
```

Comment:
I know of `dst-explorer.py` but it was too difficult/intransparent to use
with its lack of docs hence I started my own which just starts python,
opens the DST and then leaves you to do what you want to do. The Bender
script seems to do a lot of magic, takes/took "for ever" to start, had no
docs, ...

On Wed, May 20, 2015 at 1:18 PM Alex Pearce <notifications@github.com>
wrote:

> In code/dst-explore.py
> <https://github.com/betatim/lhcb-first-analysis-steps/pull/20#discussion_r30691254>
> :
>
> > @@ -0,0 +1,133 @@
> > +# SetupProject DaVinci v36r6
>
> I guess you know of dst-explorer.py script in Bender; is this script
> better?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/betatim/lhcb-first-analysis-steps/pull/20/files#r30691254>
> .
>